packages/inter-protocol/src/proposals/utils.js

```
@@ -90,6 +91,7 @@ export const makeInstallCache = async (
   { installCacheKey = 'installCache', loadBundle },
 ) => {
   /** @type {CopyMap<string, {installation: Installation, boardId: string, path?: string}>} */
+  // @ts-expect-error cast
```

Please actually cast narrowly instead of ignoring all errors on the line.